salt/metalk8s/container-engine/containerd/files/50-metalk8s.conf.j2

```diff
@@ -2,3 +2,4 @@
 # See https://github.com/containerd/containerd/issues/3201
 # See https://github.com/containerd/containerd/pull/3202
 LimitNOFILE=1048576
+Environment="{{ environment }}"
```

Comment:
Is it picking up whatever is in `/etc/environment` on its own? Or should we manually copy its contents here?

Reply:
In my understanding, it's a merge between what's already defined in environment and what we define here, that's why I only defined NO_PROXY, so that we use HTTP(S)_PROXY defined system wide.

Reply:
OK perfect.